istio.deps

```diff
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "8fece7afb77d6bd6308180d1df5d7105f73cdb45"
```

Comment:
This will be removed as it's getting covered by https://github.com/istio/istio/pull/27917